app/services/comment_notifier.rb

```diff
@@ -4,11 +4,19 @@ def initialize(broadcaster: PusherBroadcaster.new, serializer: CommentSerializer
     @serializer = serializer
   end
 
-  def call(debate, comment)
+  def call(comment, channel)
     @broadcaster.push(
-      "dashboard_channel_#{debate.code}",
+      channel,
       "comment_added",
       @serializer.new(comment).to_h
     )
   end
+
+  def send_comments(debate_id, comments)
+    @broadcaster.push(
+      "dashboard_channel_multiple_#{Debate.find(debate_id)[:code]}",
+      "comments_added",
+      comments
+    )
+  end
```

Comment:
There is a bit of inconsistency in here, `call` method accepts `comment` and `channel` attributes and `send_comments` accepts `debate_id` and serialized `comments`.

Also if we want to use this single class for both cases (sending single and multiple comments) we should rename `call` method to something like `send_comment`.

Both methods should accept same type of attribute to determine channel name (either `channel`, `debate` or `debate_code`). I think debate should not be fetched from db here